packages/core/admin/admin/src/tests/StrapiApp.test.js

```diff
@@ -4,6 +4,8 @@ import { Components, Fields } from '../core/apis';
 import StrapiApp from '../StrapiApp';
 import appReducers from '../reducers';
 
+window.matchMedia = jest.fn(() => false);
```

Comment:
I think that you can create a mock in the `admin-test-utils` so you can do this globally in our jest config instead of repeating it everywhere.